doc/editorconfig.txt

```diff
@@ -86,6 +91,21 @@ Example: Avoid loading EditorConfig for any remote files over ssh
 >
  let g:EditorConfig_exclude_patterns = ['scp://.*']
 <
+
+                                            *g:EditorConfig_exec_path*
+The file path to the EditorConfig core executable. You could set this value in
+your |vimrc| like this:
+>
+ let g:EditorConfig_exec_path = 'Path to your EditorConfig Core executable'
+<
+The default value is empty.
+
+If "g:EditorConfig_exec_path" is not set, the plugin will use the "vim_core"
+mode regardless of the setting of |g:EditorConfig_core_mode|.
+
+Changes to "g:EditorConfig_exec_path" will not take effect until Vim
+is restarted.
```

Comment:
There used to be a path search program here to search for default location:

https://github.com/editorconfig/editorconfig-vim/compare/68f8136d2b018bfa9b23403e87d3d65bc942cbc3...b03ab7a759a4f40a8f1b43c6cf065e5715e29f56#diff-b58403939457354bd5850ccec1b0c8e3L111

Do you think it's worth being added back?

Reply:
A good question.  I do not think it is worth the complication for what is now a backup option.  I suggest that a user who knows enough to acquire a core and set this option knows enough to provide the full path :) .  Removing path search also increases portability.

Reply:
Sounds good. Maybe default to `which editorconfig`?

Reply:
I can, but I must honestly tell you I have no desire to do so :) .  

- Any form of automatic search adds code and complication.
- Platform-specific code (which path search is) is harder to test and more likely to fail in edge cases.
- I am not an expert on `shellescape`.
- `which` doesn't exist on all platforms.  Vim supports MS-DOS, which has neither `which` nor `where`.

This is an option almost no-one will use.  Anyone who knows enough to use this option knows enough to add two lines to `.vimrc` instead of one.

I will update the documentation for `g:EditorConfig_core_mode` to note that you have to set `g:EditorConfig_exec_path` if you are using `external_command` mode.  What do you think about that?

Reply:
Sounds reasonable to me. Thanks!